.github/workflows/tag-release.yml

```diff
@@ -71,9 +71,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           RELEASE_NAME: ${{ steps.get_version.outputs.version }}
         run: |
+          echo "Anaconda release $RELEASE_NAME created automatically from git tag.\n\n" > release.txt
+          # lines between %changelog and an empty line | remove the first two lines
+          sed -n '/^%changelog$/,/^$/p' anaconda.spec.in | sed '1,2d' >> release.txt
```

Comment:
Hmm, I wonder if this is a good approach. The point is that we want to remove `%changelog` from the spec file. It shouldn't be there and then this code will break.
I would rather like to see call of `makedump` with some option which would only give you the changelog part and nothing else.

Reply:
See the latest changes for an approach that uses git instead.